server/config.ts

```diff
@@ -49,6 +49,7 @@ export default {
   production,
   https: process.env.NO_HTTPS === 'true' ? false : production,
   staticResourceCacheDuration: '1h',
+  logLevel: get('LOG_LEVEL', 'info'),
```

Comment:
Nice using the config object for this -- I didn't think of that for the other UIs. Might well update!